src/test/librados/watch_notify.cc

```diff
@@ -137,11 +137,14 @@ TEST_F(LibRadosWatchNotify, WatchNotify) {
   char buf[128];
   memset(buf, 0xcc, sizeof(buf));
   ASSERT_EQ(0, rados_write(ioctx, "foo", buf, sizeof(buf), 0));
+  std::cout << "object write complete" << std::endl;
```

Comment:
i'd drop this change. for a test, no news is good news.

Reply:
done